packages/core/package.json

```diff
@@ -16,30 +17,12 @@
 		"test": "vitest run",
 		"test:ui": "vitest --ui"
 	},
-	"files": [
-		"dist"
-	],
 	"exports": {
-		".": {
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
-		},
-		"./css": {
-			"types": "./dist/css.d.ts",
-			"default": "./dist/css.js"
-		},
-		"./html": {
-			"types": "./dist/html.d.ts",
-			"default": "./dist/html.js"
-		},
-		"./js": {
-			"types": "./dist/js.d.ts",
-			"default": "./dist/js.js"
-		},
-		"./parsers": {
-			"types": "./dist/parsers.d.ts",
-			"default": "./dist/parsers.js"
-		}
+		".": "./index.ts",
+		"./css": "./tooling/css/index.ts",
+		"./html": "./tooling/html/index.ts",
+		"./js": "./tooling/js/index.ts",
+		"./parsers": "./tooling/parsers.ts"
```

Comment:
we'll have to keep in mind that `@sveltejs/cli-core` will be a public package in the future, so this would have to be updated back to its original state when community add-ons are released

Reply:
> Will it be `@sveltejs/cli-core` or maybe under `sv` like `sv/core` ?

its still TBD, but our plans were to have `core` be its own standalone package so that community add-ons don't have `sv` as a direct dependency

> What about `@sveltejs/create` & `@sveltejs/addons` ?

`create` should probably stay in `sv`, but there's an argument to be made as to where `addons` should go. it may be more useful to have it be part of `core` but we can figure it out when we get there :p